include/pm/policy.h

```diff
@@ -20,11 +20,12 @@ extern "C" {
  * idle and returns the most appropriate state based on the number of
  * ticks to the next event.
  *
+ * @param cpu CPU index.
  * @param ticks The number of ticks to the next scheduled event.
  *
- * @return The power state the system should use.
+ * @return The power state the system should use for the given cpu.
  */
-struct pm_state_info pm_policy_next_state(int32_t ticks);
+struct pm_state_info pm_policy_next_state(int cpu, int32_t ticks);
```

Comment:
At the risk of sounding like a broken record, again, I would suggest using an unsigned type over a signed type. I will also refrain from repeating this same comment in subsequents commits in this PR--if applicable.

It is also not a hill that I am willing to die on. :)